src/envelope.rs

```diff
@@ -88,20 +88,20 @@ impl Instrumentation {
     }
 
     #[cfg_attr(not(feature = "instrumentation"), allow(unused_variables))]
-    fn started<A>(message: &'static str) -> Self {
+    fn started<A>(message_type: &'static str) -> Self {
         #[cfg(feature = "instrumentation")]
         {
             let parent = Span(tracing::debug_span!(
                 "xtra_actor_request",
-                actor = std::any::type_name::<A>(),
-                %message,
+                actor = %std::any::type_name::<A>(),
```

Comment:
For consistency, should we rename this to `actor_type`?

Reply:
Good point